third-party/unittest/googletest/README.LLVM

```diff
@@ -1,22 +1,21 @@
 LLVM notes
 ----------
 
-This directory contains Google Test 1.10.0, with all elements removed except for
-the actual source code, to minimize the addition to the LLVM distribution.
+This directory contains Google Test 1.14.0,
+revision `f8d7d77c06936315286eb55f8de22cd23c188571`, with all elements removed
```

Comment:
We don't include the revision in the `googlemock` `README.LLVM` file so I'd avoid it here also for consistency (or include it in the other one as well).

Reply:
Parts of the googletest documentation claim to follow a "live at HEAD" philosophy, so a hash reference (here and in the other README) seems appropriate. Although using a tag, if there is one, would be preferable.

Reply:
v1.14.0 is a valid tag and `f8d7d77c06936315286eb55f8de22cd23c188571` is the revision that it resolves to. The content matches the 1.14.0 source tar ball. The reason that I include a hash here is because the previous roll v1.10.0 doesn't have a valid tag and I had a bit hard time to find the correct revision for doing a diff to see what LLVM have patched. Since v1.14.0 is a valid tag, I think it is appropriate to remove the hash and just leave v1.14.0 in the doc.